packages/frontend/src/pages/Dashboard.jsx

```diff
@@ -0,0 +1,222 @@
+import React from 'react'
+import { observer } from 'mobx-react-lite'
+import './dashboard.css'
+import Button from '../components/Button'
+import Tooltip from '../components/Tooltip'
+
+import User from '../contexts/User'
+
+export default observer(() => {
+  const userContext = React.useContext(User)
+  const [remainingTime, setRemainingTime] = React.useState(0)
+  const [reqRep, setReqRep] = React.useState({})
+  const [repProofInputs, setRepProofInputs] = React.useState({})
+  const [repProof, setRepProof] = React.useState(null)
+
+  const updateTimer = () => {
+    if (!userContext.userState) {
+      setRemainingTime('Loading...')
+      return
+    }
+    const time = userContext.userState.calcEpochRemainingTime()
+    setRemainingTime(time)
+  }
+
+  React.useEffect(() => {
+    setInterval(() => {
+      updateTimer()
+    }, 1000)
+  }, [])
+
+  React.useEffect(() => {
+    if (!userContext.userState) {
+      setTimeout(() => {
+        userContext.epochKey(reqRep.nonce ?? 0).then((key) => setReqRep((v) => ({ ...v, epochKey: key })))
+      }, 1000)
+    } else {
+        userContext.epochKey(reqRep.nonce ?? 0).then((key) => setReqRep((v) => ({ ...v, epochKey: key })))
+    }
+  }, [reqRep.nonce])
+
+  if (!userContext.userState) {
+    return (
+      <div className="container">
+        Loading...
+      </div>
+    )
+  }
+
+  return (
+    <div>
+      <h1>Dashboard</h1>
+      <div className="container">
+        <div className="info-container">
+          <div className='info-item'>
+            <h3>Epoch</h3>
+            <Tooltip text='This is info about the epoch.'/>
+          </div>
+          <div className='info-item'>
+            <div>Current epoch #</div>
+            <div className='stat'>{userContext.userState?.calcCurrentEpoch()}</div>
+          </div>
+          <div className='info-item'>
+            <div>Remaining time</div>
+            <div className='stat'>{remainingTime}</div>
+          </div>
+          <div className='info-item'>
+            <div>Latest transition epoch</div>
+            <div className='stat'>{userContext.latestTransitionedEpoch}</div>
+          </div>
+
+          <hr/>
+
+          <div className='info-item'>
+            <h3>Latest Reputation</h3>
+            <Tooltip text='This is info about latest reputation.'/>
+          </div>
+          <div className='info-item'>
+            <div>Positive</div>
+            <div className='stat'>{userContext.reputation.posRep?.toString()}</div>
+          </div>
+          <div className='info-item'>
+            <div>Negative</div>
+            <div className='stat'>{userContext.reputation.negRep?.toString()}</div>
+          </div>
+          <div className='info-item'>
+            <div>Graffiti</div>
+            <div className='stat'>0x{userContext.reputation.graffiti?.toString(16)}</div>
```

Comment:
This overflows the container: 
<img width="482" alt="image" src="https://user-images.githubusercontent.com/631020/206244828-253fc687-fabc-4fd9-87f0-c0febc6a7b5d.png">

Maybe set `work-break: all`

Reply:
<img width="1362" alt="Screen Shot 2022-12-07 at 11 39 36 AM" src="https://user-images.githubusercontent.com/79813968/206267767-54662e66-ebb0-4060-acbd-fa40e75f47f3.png">
I'm getting this error on requesting rep, so I'm not able to get a graffiti value on my screen.

Reply:
Right, this is almost certainly the race condition documented in #7. Redeploy your contracts and restart the relay+frontend, or clear your localstorage to make a new identity

Reply:
got it thanks.
✅ and fixed